source/backend/dal/Repositories/LeasePaymentRepository.cs

```diff
@@ -47,40 +47,10 @@ public IEnumerable<PimsLeasePayment> GetAllTracking(DateTime startDate, DateTime
             return this.Context.PimsLeasePayments
                 .AsSplitQuery()
                 .Include(p => p.LeasePaymentCategoryTypeCodeNavigation)
-                .Include(p => p.LeasePeriod)
-                    .ThenInclude(t => t.Lease)
-                    .ThenInclude(p => p.PimsPropertyLeases)
-                    .ThenInclude(p => p.Property)
-                    .ThenInclude(p => p.PimsHistoricalFileNumbers)
-                    .ThenInclude(h => h.HistoricalFileNumberTypeCodeNavigation)
-                .Include(p => p.LeasePeriod)
-                    .ThenInclude(t => t.Lease)
-                    .ThenInclude(p => p.RegionCodeNavigation)
-                .Include(p => p.LeasePeriod)
```

Comment:
This whole query is a "tracking" query - but it is only used to generate a report (which is read-only). 
Any considerations to add "AsNotTracking" to the query so EF is not tracking these entities wastefully?